community/kernel/src/main/java/org/neo4j/graphdb/factory/GraphDatabaseSettings.java

```diff
@@ -240,6 +240,10 @@
     @Description("The maximum time interval of a transaction within which it should be completed.")
     public static final Setting<Long> transaction_timeout = setting( "dbms.transaction.timeout", DURATION, String.valueOf( UNSPECIFIED_TIMEOUT ) );
 
+    @Description("Configures the time interval between transaction monitor checks. Determines how often " +
+            "monitor thread will check transaction for timeout.")
+    public static final Setting<Long> transaction_monitor_check_interval = setting( "dbms.transaction.monitor.check.interval", DURATION, "10s" );
```

Comment:
duration setting type does not allow negative values already,
do not know what would be the real use case for that but why not....just in case something goes really wrong...will not schedule job if 0 is specified then